joint_limits:
  pan_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  right_arm_tilt_joint:
    has_velocity_limits: true
    max_velocity: 0.5
    has_acceleration_limits: true
    max_acceleration: 1
  left_arm_wrist_yaw_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  left_arm_gripper_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  left_arm_tilt_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  left_arm_lift_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  left_arm_rotate_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  right_arm_rotate_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  left_arm_elbow_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  left_arm_wrist_tilt_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  right_arm_wrist_yaw_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  right_arm_lift_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  right_arm_wrist_tilt_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  right_arm_elbow_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
  right_arm_gripper_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
